common/setups/serialization.py

```diff
@@ -348,6 +349,62 @@ def _sis_hash(self):
         return sis_hash_helper(self.hash)
 
 
+class Call(SerializerObject):
+    """
+    SerializerObject that serializes the call of a callable with given arguments.
+    The return values of the call are optionally assigned to variables of a given name.
+    Example:
+    Call(callable_name="range", args=[1, 10], kwargs=[("step", 2)], return_assign_variables="number_range")
+    ->
+    number_range = range(1, 10, step=2)
+    """
+
+    def __init__(
+        self,
+        callable_name: str,
+        args: Optional[List[Union[str, DelayedBase]]] = None,
+        kwargs: Optional[List[Tuple[str, Union[str, DelayedBase]]]] = None,
```

Comment:
I would allow only for kwargs, but add an extra parameter for non-hashed kwargs in case you want to have that.